src/MockClassifier.Api/Controllers/MessagesController.cs

```diff
@@ -0,0 +1,97 @@
+﻿using Microsoft.AspNetCore.Mvc;
+using MockClassifier.Api.Interfaces;
+using MockClassifier.Api.Models;
+using MockClassifier.Api.Services.Dmr;
+using System.Text;
+using System.Text.Json;
+
+namespace MockClassifier.Api.Controllers
+{
+    [Route("/dmr-api/messages")]
+    [ApiController]
+    public class MessagesController : ControllerBase
+    {
+        private readonly IDmrService _dmrService;
+        private readonly ITokenService _tokenService;
+        private readonly INaturalLanguageService _naturalLanguageService;
+        private readonly IEncodingService _encodingService;
+
+        public MessagesController(IDmrService dmrService, ITokenService tokenService, INaturalLanguageService naturalLanguageService, IEncodingService encodingService)
+        {
+            _dmrService = dmrService;
+            _tokenService = tokenService;
+            _naturalLanguageService = naturalLanguageService;
+            _encodingService = encodingService;
+        }
+
+        /// <summary>
+        /// Processes a string to identify classifications and issues call backs to Dmr for each classification.
+        /// </summary>
+        /// <param name="input">Property which contains the base64 encoded payload containing the message to be classified</param>
+        /// <returns>An empty 202/Accepted result</returns>
+        [HttpPost]
```

Comment:
```suggestion
        [HttpPost]
        [ProducesResponseType(StatusCodes.Status202Accepted)]
        [ProducesResponseType(StatusCodes.Status400BadRequest)]
```

@yahya130 suggested this for the Dmr review.